test/unit/strpak.unit.cpp

```diff
@@ -28,3 +28,25 @@ TEST(strpak, compare_case_insensitive_string)
 	EXPECT_LT( _strnicmp(long_string_upper_case, long_string_wrong_case, 22), 0);
 	EXPECT_EQ( _strnicmp(long_string_upper_case, long_string_wrong_case, 10), 0);
 }
+
+TEST(strpak, convert_case_functions)
+{
+	// Set up strings
+	char uppercaseString[] = "this should be uppercase.";
+	char lowercaseString[] = "THIS SHOULD BE LOWERCASE.";
+	char randomCaseString[] = "Hello There ARE many case HERE!";
+	char numberString[] = "There are 12311 numbers (#).";
```

Comment:
Should we also test number/letter mixed words, as you sometimes see in variable names, e.g. "ASHRAE205"